backend/app/services/memory/manager.py

```
@@ -0,0 +1,377 @@
+                        "Failed to delete memory %s: %s", memory_id, e, exc_info=True
+                    )
+                    error_count += 1
+
+            logger.info(
+                "Cleaned up %d memories for task %d (%d errors)",
+                delete_count,
+                task_id,
+                error_count,
+            )
+
+            return delete_count
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)